samples/MvvmSampleUwp/Views/AsyncRelayCommandPage.xaml.cs

```diff
@@ -14,5 +16,11 @@ public sealed partial class AsyncRelayCommandPage : Page
     public AsyncRelayCommandPage()
     {
         this.InitializeComponent();
+
+        ViewModel = Ioc.Default.GetRequiredService<AsyncRelayCommandPageViewModel>();
+
+        DataContext = ViewModel;
     }
+
+    public AsyncRelayCommandPageViewModel ViewModel { get; }
```

Comment:
We can avoid the second state, all these should just be cases over `DataContext`.
As in:
```csharp
// Constructor
DataContext = Ioc.Default.GetRequiredService<AsyncRelayCommandPageViewModel>();

// Property
public AsyncRelayCommandPageViewModel ViewModel => (AsyncRelayCommandPageViewModel)DataContext;
```

Reply:
fixed